### DISCUSSION TITLE
a11y: Are there any plans to support developers in creating accessible web apps?

### DISCUSSION THREAD

**Discussion by andorfermichael** [nuxt/nuxt#9313] [Ideas]
I have done some research and I could not find any results related to nuxt and a11y. I could only find an initiative for vue called [vue-a11y](https://vue-a11y.com/), unfortunately, their progress seems to be relatively slow. I was wondering if the nuxt team has planned something similar and/or working together with the team of vue-a11y. In my opinion, the subject gets too little attention, unfortunately.

**Comment by danielroe** [comment]
@andorfermichael `vue-a11y/vue-axe` works well with Nuxt: see [their docs](https://axe.vue-a11y.com/guide/#nuxt-js) for how to integrate. Much of the work in making sure you're building an accessible web app is independent of framework but we'd certainly welcome suggestions!

**Reply by andorfermichael** [reply]
You are right that most of the tasks related to accessibility are framework-independent. However, I think frameworks like nuxt/vue could support developers etc. [vue-a11y](https://vue-a11y.com/) is a step in the redirect, but it seems they don't have much resources and power. Thanks for clarifying anyway. ;)

**Comment by kissu** [comment]
Not sure if you're looking for a UI lib with accessible components, if it's the case you could try Chakra: https://vue.chakra-ui.com/with-nuxt

**Reply by andorfermichael** [reply]
Thanks for sharing. However, I use tailwindcss and a custom component set and I don't think it's a good idea to add another lib. Nevertheless, maybe I can use Chakra components' accessibility as a template for my own components.

**Reply by kissu** [reply]
Yes, I do also use Tailwind and was expecting to somehow remove the styling of Chakra to just keep the ARIA and related things to a11y as asked in [this Github issue](https://github.com/chakra-ui/chakra-ui-vue/issues/427).

At some point, I'll learn how to do a11y in a vanilla way I guess.

**Reply by andorfermichael** [reply]
oh, great, thanks for that issue.
